### ISSUE TITLE
[Feature Request] Multi-mode itinerary planning

### ISSUE THREAD

**Issue by pelderson** [vmarc/knooppuntnet#215]
I have received several requests for multi-mode planning, i.e. to plan a route consisting of (e.g.) a hiking section along the hiking network, then a canoe section along the canoo network, then bicycle along the bicycle network, With Yves I discussed how this could work for the user, in the Planner.  This is what we came up with. 
----
The idea would be to plan one piece, then switch transport but carry over the result so far, and continue using the new transport mode  etc. 

I think there should be an option for the user to see the complete multi-transport route, maybe a preview could suffice? 

And, it should be possible to have some distance between, say, the end node of you riding section and the start node of your canooing section. They should not be required to use the same connection node. 

To sum up: 
1. carry over the itinerary to different transport mode; 
2. optional multi-mode preview;
3. discontinuous itinerary.
----

We think this can be achieved in Planner using current tagging.
(For me, it counts that this would really put Knooppuntnet Planner ahead of all other Node Network planners, a unique functional plus)
